pages/api/query.js

```diff
@@ -6,6 +6,11 @@ const conditionalLink = `
   "externalLink": linkExternal
 `;
 
+// ArrayOfTitleAndText[]{
```

Comment:
I think these commented lines should be removed in order to avoid confusion. The schema field array name is `arrayOfTitleAndText`.